packages/vite/src/node/__tests__/env.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { posix } from 'node:path'
+import { fileURLToPath } from 'node:url'
+import { describe, expect, test } from 'vitest'
+import { loadEnv } from '../env'
+
+const __dirname = fileURLToPath(new URL('.', import.meta.url))
+
+describe('loadEnv', () => {
+  test('basic', async () => {
```

Comment:
```suggestion
  test('basic', () => {
```
Maybe we could remove these `async`s?